styles/modal.css

```diff
@@ -14,31 +10,35 @@
   border-radius: 10px 10px 0 0 !important;
 }
 
-@media screen and (min-width: 768px) {
-  .react-modal-sheet-container {
-    width: 400px !important;
-    max-height: calc(90vh - 2vw) !important;
-    height: max-content !important;
-    overflow-y: auto;
-    overflow-x: hidden;
-    right: 8px !important;
-    left: auto !important;
-    top: 8px !important;
-  }
+/*.react-modal-sheet-backdrop {*/
```

Comment:
nit:

Let's remove this comments. If anything, we will find them in history :)

Reply:
nit:

Let's remove this comments. If anything, we will find them in history :)

Reply:
nit:

Let's remove CSS-comments. If anything, we will find them in history